systems/plants/@RigidBodyManipulator/forwardKin.m

```diff
@@ -1,4 +1,4 @@
-function [x,J,dJ] = forwardKin(obj,kinsol,body_or_frame_ind,pts,rotation_type, base_or_frame_ind)
+function [x,J,dJ] = forwardKin(obj,kinsol,body_or_frame_ind,pts,rotation_type, base_or_frame_ind, options)
```

Comment:
since we're pulling the trigger on the options structure, should we move rotation_type and base_or_frame_ind into that, too?  (also, could we make it base_or_frame_id)